bin/aurelia.js

```diff
@@ -56,17 +56,15 @@ function handleCommands(env) {
   }
 
   aurelia.init({
-    env : env
+      env : env
+    , argv: argv
   });
 
-  if (env.configPath) {
+  if (env.configPath && env.modulePath) {
     require(env.configPath);
```

Comment:
@joelcoxokc We have resolved these in the previous PR. And as far as I know It's working correctly. Suppose when use executes `new` command it does not have the `moduelPath` so the global version is being executed. Here is the code from the `master` branch:

```javascirpt
if (!env.modulePath) {
    logger.log('warn', 'Local aurelia installation not found!');
    aurelia = require('../index');
  } else {
    aurelia = require(env.modulePath);
  }
```